src/components/organisms/UiHorizontalPaging/stories/MobileMenu.vue

```diff
@@ -100,8 +100,17 @@ import { focusElement } from '../../../../utilities/helpers';
 
 defineOptions({ inheritAttrs: false });
 const attrs = useAttrs();
-const args = computed(() => (attrs));
-const value = inject('value', []);
+const args = computed(() => (Object.keys(attrs).reduce((object, key) => {
```

Comment:
You use this twice already, maybe make it a helper? 🤔 
I know it's simple function, but it's a bit lengthy. 😛